packit/api.py

```diff
@@ -226,10 +226,38 @@ def copr_helper(self) -> CoprHelper:
 
     @property
     def sync_release_env(self):
-        return {
-            "PACKIT_DOWNSTREAM_REPO": str(self.dg.local_project.working_dir),
-            "PACKIT_UPSTREAM_REPO": str(self.up.local_project.working_dir),
-        } | self.up.package_config.get_package_names_as_env()
+        if self.config.command_handler == RunCommandType.sandcastle:
+            # import this here, we don't want to depend upon
+            # sandcastle and python-kube if not in service
+            from sandcastle.constants import SANDCASTLE_EXEC_DIR
+
+            # working dirs should be placed under
+            # self.config.command_handler_working_dir
+            # when running this code as a service
+            downstream_suffix = str(self.dg.local_project.working_dir).replace(
+                self.config.command_handler_work_dir + "/", ""
+            )
+            upstream_suffix = str(self.up.local_project.working_dir).replace(
+                self.config.command_handler_work_dir + "/", ""
+            )
```

Comment:
If I get correctly what this is supposed to do, I think this is much more clear and self-explanatory:

```suggestion
            downstream_suffix = Path(self.dg.local_project.working_dir).relative_to(
                self.config.command_handler_work_dir
            )
            upstream_suffix = Path(self.up.local_project.working_dir).relative_to(
                self.config.command_handler_work_dir
            )
```